webapp/src/main/webapp/vue-apps/Admin/components/AdminApp.vue

```diff
@@ -77,6 +77,11 @@ export default {
       log: null
     };
   },
+  computed: {
+    externalVideoConnector() {
+      return this.providersConfig.length > 0 && this.providersConfig.filter(provider => provider.title === 'ExternalVideo')[0];
+    }
+  },
```

Comment:
you cannot have reference to externalVisio connector here. 
you are in the module webconferencing, which should not depends of connectors